### PR TITLE
Updating CODEOWNERS for Communication

### PR DESCRIPTION
**Pull request by petrsvihlik**
# Contributing to the Azure SDK

- Code owners updated to my best knowledge. 

Reviewers, please confirm.

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -193,10 +193,10 @@
 #/<NotInRepo>/            @ms-premp @qiaozha
 
 # PRLabel: %Communication - Calling Server
-/sdk/communication/Azure.Communication.CallingServer/   @Azure/acs-identity-sdk @petrsvihlik @AikoBB @maximrytych-ms @martinbarnas-ms
+/sdk/communication/Azure.Communication.CallingServer/
```
Comment (petrsvihlik) [inline .github/CODEOWNERS:196]: Information requested from the former owners.

**Review by jsquire — APPROVED** [review]

**Comment by petrsvihlik** [comment]
> @petrsvihlik Petr Svihlik FTE, I believe @[rtrevino89](https://github.com/rtrevino89) Roberto Trevino FTE shouldn't be in this list as he does not work with the C# SDK.

thanks! updated!

**Review by guilhermeluizsp — APPROVED** [review]

**Review by rtrevino89 — APPROVED** [review]

**Pull request merged**